index.js

```diff
@@ -29,26 +29,33 @@ const {
 const input = cli.input;
 const flags = cli.flags;
 const { clear, debug } = flags;
+const path = require(`path`);
+const { setConfig } = require(`./utils/config`);
+const process = require(`process`);
 
 (async () => {
 	init({ clear });
 	input.includes(`help`) && cli.showHelp(0);
 	debug && log(flags);
 
+	let projectPath = ``;
+
 	const useQuickStart = input.includes(`new`) ? quickStart(flags) : false;
 	try {
 		await detectDownloadsAndStars();
 		await detectProjectType();
 		await detectPackageManager();
 		if (!(await detectStrapiProject())) {
-			await createStrapiProject();
+			projectPath = await createStrapiProject();
 		}
 
 		if (input.includes(`reset`)) {
 			await reset();
 			goodbye();
 			return;
 		}
+		process.chdir(projectPath);
```

Comment:
if we are only using this in `detecthStrapiProject()` can't we then do something like this ?

```javascript
if (!(await detectStrapiProject())) {
			const projectPath = await createStrapiProject();
process.chdir(projectPath);
setConfig({outDir: path.join(process.cwd())});
}
```
Not sure but I think we can ?

Reply:
made the change and moved the code inside if condition